lib/image_optim.rb

```diff
@@ -37,6 +38,12 @@ class ImageOptim
   # Allow lossy workers and optimizations
   attr_reader :allow_lossy
 
+  # Cache directory
+  attr_reader :cache_dir
+
+  # Cache workers digest
+  attr_reader :cache_workers
```

Comment:
`cache_workers` name seems not descriptive. I did not find some obvious name, but something like `cache_worker_digests` or `cache_by_digest`, what do you think?